source/transactions/tests/abort.json

```diff
@@ -8,12 +8,11 @@
       "operations": [
         {
           "name": "startTransaction",
-          "arguments": {
-            "session": "session0"
-          }
+          "object": "session0"
```

Comment:
I know order doesn't matter, but thinking in an object oriented fashion I would put the "object" first:

    "object" : "session0",
    "name" : "startTransaction"

maps to:

    session0.startTransaction(...)

But either order works for me.

Reply:
I agree but I also think the tests are easier to read when the method name is first so I going to keep the current order.